connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoReader.scala

```diff
@@ -68,6 +75,51 @@ private[kusto] object KustoReader {
                                           partitionInfo: KustoPartitionParameters,
                                           options: KustoReadOptions,
                                           filtering: KustoFiltering): RDD[Row] = {
+    var paths: Seq[String] = Seq()
+    // if distributedReadModeTransientCacheEnabled is set to true, then check if path is cached and use it
+    // if not export and cache the path for reuse
+    if (options.distributedReadModeTransientCacheEnabled) {
+      val key = DistributedReadModeTransientCacheKey(request.query, request.kustoCoordinates, request.authentication)
+      if (distributedReadModeTransientCache.contains(key)) {
+        KDSU.logInfo(myName, s"distributedReadModeTransientCache: ${options.distributedReadModeTransientCacheEnabled} cache: hit, reusing cached export paths")
```

Comment:
options.distributedReadModeTransientCacheEnabled is always true here - same for the other logs .. the value is already know )
```suggestion
        KDSU.logInfo(myName, s"Fetching from distributedReadModeTransientCache: hit, reusing cached export paths")
```

Reply:
updated, removed the else block log as exportToStorage will log exporting log.